scripts/build_coverage.sh

```diff
@@ -92,8 +92,39 @@ if [ "$skip_gn" == false ]; then
     ninja -C "$OUTPUT_ROOT" check
 fi
 
+echo "PWD:$PWD"
+echo "CHIP_ROOT:$CHIP_ROOT"
+echo "OUTPUT_ROOT:$OUTPUT_ROOT"
+
+rm -rf "$OUTPUT_ROOT/obj/src/access/tests"
```

Comment:
This seems to be somewhat hard to maintain. Could we use something like `find "$OUTPUT_ROOT/obj/src/" -name 'tests' -type d` ?